# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../scanlogd/makefile.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-03-05 11:03:15.000000000 +0100
+++ ./Makefile	2007-08-08 15:18:50.000000000 +0200
@@ -1,8 +1,6 @@
 CC = gcc
 LD = $(CC)
 RM = rm -f
-CFLAGS = -Wall -O2 -fomit-frame-pointer
-LDFLAGS = -s
 
 PCAP_H = -I/usr/include/pcap
 PCAP_L = -lpcap
